### PR TITLE
Add DockerHub Descriptions for the gateway containers

### LINKED ISSUE #359: Docker Readmes
We need docker readmes for all of the docker images

- [ ] Account
- [ ] Graph
- [ ] Content Publisher
- [ ] Content Watcher

image

### PR DESCRIPTION
**Pull request by mattheworris**
# Problem

Gateway docker container images do not have a description in DockerHub.

Closes #359

# Solution

- Update the release.yml to publish the overview description
- Add the overview.md files.

### CODE REVIEW

File: Docker/account-service.overview.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Account Service Overview
+
+Account Service is a service enabling easy interaction with [DSNP](https://dsnp.org/) accounts on [Frequency](https://docs.frequency.xyz/). Accounts can be defined as a user's handle and MSA Id.
```
Comment (wilwade) [inline Docker/account-service.overview.md:3]: Can be is an odd phrasing. Trying out a different way to say it.
```suggestion
Account Service is a service enabling easy interaction with [DSNP](https://dsnp.org/) accounts on [Frequency](https://docs.frequency.xyz/). Accounts are identified by the Frequency MSA Id.
```

**Review by wilwade — DISMISSED** [review]
One small specific change. Generally looks good.

Question I came away with was "Ok. How do I run this?" Perhaps we need to add an example to the top and move up the env vars section?

Or to put it another way. How can we get someone enough information to run it front and center.

File: Docker/account-service.overview.md (hunk 1 above)
Comment (mattheworris) [reply Docker/account-service.overview.md:3]: Yeah, the summaries are copied directly from the `developer-docs/README.md`. We may want to keep those in sync, or a lot of that needs to go into the "Live Docs"

**Review by JoeCap08055 — APPROVED** [review]
Nice! I had always wondered how to do this in a pipeline.

File: docs/src/Run/AccountService/RunAccountService.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Install NPM Dependencies:
+
+```sh
+  npm install
+```
+
+## Usage
+
+Note: using [docker compose file](../../docker-compose.yaml) to start the services. This will start the services in development mode.
+
```
Comment (jeanettedepatie) [inline docs/src/Run/AccountService/RunAccountService.md:34]: Recommend replacing with:
"Note: Using [docker compose file](../../docker-compose.yaml) to start the services will start the services in development mode."

File: docs/src/Run/AccountService/RunAccountService.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+In order to run this project you need:
+
+- [Node.js](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
+
+## Environment Variables
+
+Modify any environment variables in the `.env` file as needed. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](../../env-files/account.template.env).
+
```
Comment (jeanettedepatie) [inline docs/src/Run/AccountService/RunAccountService.md:14]: Note: these links don't work for me.  Probably a permissions thing.

File: Docker/content-watcher.overview.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+The Content Watcher Service is part of the [Gateway](https://projectlibertylabs.github.io/gateway) collection of services that provides a familiar callback API to retrieve content and publishing announcements from the [Frequency](https://docs.frequency.xyz/) chain. The service handles all of the necessary blockchain interaction and allows clients to interact using a familiar, Web2-friendly interface.
+
+## Key Features
+
+### API
+
+- **Parse DSNP Messages on Frequency**
+- **Send content to registered webhooks**
+- **Simple Schema and MSA Id based filtering**
+
```
Comment (jeanettedepatie) [inline Docker/content-watcher.overview.md:12]: Note: Picky point, but it's better if these bullets are formatted the same.  So for the third bullet maybe:
"Filter via simple Schema and MSA Id-based filtering"

File: Docker/graph-service.overview.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Graph Service Overview
+
+The Graph Service is a service enabling easy interaction with [DSNP](https://dsnp.org/) private and public graphs on [Frequency](https://docs.frequency.xyz/).
+
+## Key Features
+
+- **Fetch Graphs**
+- **Update Graph**
+- **Watch Graphs**
+
```
Comment (jeanettedepatie) [inline Docker/graph-service.overview.md:10]: Should it be "Update Graphs" plural to match the rest?

File: docs/src/Run/AccountService/RunAccountService.md (hunk 3 above)
Comment (mattheworris) [reply docs/src/Run/AccountService/RunAccountService.md:14]: I'll double check them, but they will not be valid until the changes are published.

**Review by wilwade — COMMENTED** [review]
Links need to be absolute instead of relative for the links into github files, otherwise looks good.

File: Docker/content-publishing-service.overview.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+### API
+
+- **Upload asset files**
+- **Create a broadcast(post)**
+- **Create a reply**
+- **Create a reaction**
+- **Update existing content**
+- **Delete existing content (tombstones)**
+- **Update a user profile**
```
Comment (saraswatpuneet) [inline Docker/content-publishing-service.overview.md:15]: should we mention about pinning to IPFS as a feature

**Review by saraswatpuneet — APPROVED** [review]
read through docker docs, one comment but looks clean and nice

**Review by wilwade — APPROVED** [review]
Let's merge it.

We'll want to focus on https://github.com/ProjectLibertyLabs/gateway/issues/630 next with (added as notes) these criteria:


- Consider the "Builder" to be the primary audience of the docs
- Developer related docs can be moved out of the mdBook docs directory and instructions placed in the repo readme or the readme for each service and just linked from mdBook.

**Pull request merged**